setup.cfg

```diff
@@ -11,7 +11,7 @@ project_urls =
     Tracker = https://github.com/DemocracyClub/dc_logging/issues
 
 [options]
-python_requires = >=3.8
+python_requires = >=3.11
```

Comment:
It looks like there are 2 places we care about using this package:

1. In our applications (which are all now running on python 3.12, across the board)
2. In AWS CloudShell.

TIL AWS CloudShell. TI also L that the most recent python version available in the Amazon Linux 2023 repository is still python 3.11. Obviously there are other ways to get python on there (pyenv, uv, etc) but it seems like the lowest friction thing to do here is say we are compatible with >=3.11 and leave the instructions in https://github.com/DemocracyClub/dc_logging/blob/main/dc_logging_legacy_import/README.md#running-from-aws-cloudshell as they are for now.

I think we can live with the test suite only being run against 3.12 though. Does that seem right to you?